test/cli.js

```diff
@@ -31,9 +31,15 @@
     expect: { timeout: true }
   },
   {
-    args: ["run", "test/fixtures/fail.ts"],
+    args: ["run", "test/fixtures/ts-error1/fail.ts"],
```

Comment:
Why was this changed?

Reply:
Because ts-loader will throw errors for all ts files in the project, such that both error messages become the error message for the one file otherwise!

I kind of see this as something we should address in https://github.com/zeit/ncc/issues/362.